tools/cldr-code/src/test/java/org/unicode/cldr/test/TestCheckAnnotations.java

```diff
@@ -0,0 +1,31 @@
+package org.unicode.cldr.test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class TestCheckAnnotations {
+
+    @ParameterizedTest
+    @ValueSource(
+            // all of these should fail
+            strings = {
+                // Soures as noted.
+
+                // zu
+                "E13.1\u2013128",
+                "E13.1\u2013127 | iphesenti",
+                "ampersand | E13.1\u2013125",
```

Comment:
fails if the code is present anywhere.

Reply:
We have special code in the tests on the server so that we can skip a test when integrating. So we could do that.
On the other hand, these might be simpler to change in XML. Is it easy to get a sense of the magnitude: eg a printout of the Console Check with -e?

Reply:
The PR run https://github.com/unicode-org/cldr/actions/runs/5661231695/job/15339288080?pr=3119 shows all of the errors.  True, I could just fix them in XML (drop all err entries, delete the error keyword if present). I'll do that.